### PR TITLE
Open last active cluster view when switching between workspaces

### LINKED ISSUE #827: Dashboard doesn't refresh after changing workspace
**Describe the bug**
When changing from one workspace to another the dashboard is "blank" (there is no current cluster shown) 

**To Reproduce**
1. With two workspaces, click on workspace widget (bottom left) and change to the other workspace
2. Observe that the workspace is shown, with no active cluster(even if there was an active cluster when the workspace was last selected)
3. Click on a cluster icon to cause the refresh

**Expected behavior**
When toggling between workspaces the dashboard should show previous snapshot of the workspace
**Screenshots**

wsRefresh

**Environment (please complete the following information):**
- Lens Version: 3.6.0-rc.1
- OS: OSX
- Installation method (e.g. snap or AppImage in Linux): dmg

### PR DESCRIPTION
**Pull request by stevejr**
This enhancement will store the last active cluster used in a workspace. If you then switch to a different
workspace and switch back the cluster view will be restored.

If a cluster is disconnected then it will not become active after switching back to the workspace - only an active cluster in the workspace will be "re-activated" when switching back to the workspace.

Signed-off-by: Steve Richards <srichards@mirantis.com>

fixes #827

### CODE REVIEW

File: src/renderer/components/cluster-manager/clusters-menu.tsx
Comment (aleksfront) [inline src/renderer/components/cluster-manager/clusters-menu.tsx]: We can use observable directly `workspaceStore.currentWorkspaceId` in every place where `workspaceStore.currentWorkspace.id` located.

**Review by aleksfront — APPROVED** [review]
LGTM!

File: src/renderer/components/cluster-manager/clusters-menu.tsx
Comment (stevejr) [reply src/renderer/components/cluster-manager/clusters-menu.tsx]: Good catch! Will update and commit

File: src/renderer/components/cluster-manager/clusters-menu.tsx
Comment (nevalla) [inline src/renderer/components/cluster-manager/clusters-menu.tsx:32]:
```suggestion
    if (clusterId) { 
```

**Review by nevalla — COMMENTED** [review]
Are you able to reproduce integration test failures locally?

**Comment by stevejr** [comment]
@nevalla - Please do not merge this PR yet as myself and Jim are getting a complete test failure when trying to test the PR branch locally.

**Comment by stevejr** [comment]
@nevalla - I am not sure why the Mac build failed. I have run it locally and it is failing at the test `shows ->Cluster page` but I think @jim-docker is looking at that. Everything else passes locally on my Mac build.

This should be refactored/merged after #1005 has been merged in due to changes in the navigation logic

**Comment by aleksfront** [comment]
Closed in #1444

**Pull request closed without merging**
